### ISSUE TITLE
Display the currency in full name not the short version like Icelandic krona not ISK

### ISSUE THREAD

**Issue by JuliusSaik** [madisvain/exchangeratesapi#86]

**Comment by Spomky** [comment]
I am not sure this is a good idea. Most of the projects I manage are in French or Japanese and this feature will increase the response size for nothing.
Moreover, there are lots of libraries in almost all programming languages that can convert the names for you.

**Comment by JuliusSaik** [comment]
hmm understood. If it's not too much trouble can you recommend a library for this sort of thing?

**Comment by Spomky** [comment]
Yes sure. For the backend (PHP), we use the [`Currencies` class from `symfony/intl`](https://symfony.com/doc/current/components/intl.html#currencies) component:

```php
use Symfony\Component\Intl\Currencies;

\Locale::setDefault('ja-JP');
$currency = Currencies::getName('USD'); //returns "米ドル"
```

For the frontend (JS), [`Numberformat`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/NumberFormat) from the Intl API (polyfill exists if [not supported by your customer browsers](https://caniuse.com/#search=intl)).
It is a bit more tricky as we have to find the correct part that corresponds to the currency name.
But nothing really complicated.

```js
const formatter= new Intl.NumberFormat('ja-JP', { style: 'currency', currency: 'EUR', currencyDisplay: 'name' });

console.log(formatter.formatToParts(0).filter(item => item.type === 'currency')[0].value); // returns "ユーロ"
```


*[EDIT]*

In your case with `ISK`:

```js
const formatter= new Intl.NumberFormat('en-US', { style: 'currency', currency: 'ISK', currencyDisplay: 'name' });
console.log(formatter.formatToParts(0).filter(item => item.type === 'currency')[0].value); // returns "Icelandic krónur"
```

**Comment by JuliusSaik** [comment]
Thanks a lot! What about java? I'm coding this in Android Studio.

**Comment by Spomky** [comment]
I am sorry but I don't use Java.
I found [this resource](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/util/Currency.html#getName%28java.util.Locale,%20int,%20boolean[]%29) that could be helpful

> String | getDisplayName(Locale locale) Returns the display name for this currency in the given locale.

**Comment by JuliusSaik** [comment]
Thank you! This will come in handy

**Comment by JuliusSaik** [comment]
ok yeah doesnt work
